epochStart/metachain/systemSCs.go

```diff
@@ -325,7 +328,12 @@ func (s *systemSCProcessor) unStakeNodesWithNotEnoughFunds(
 		return 0, err
 	}
 
-	return uint32(len(nodesToUnStake)), nil
+	nodesToStakeFromWaiting := uint32(len(nodesToUnStake))
```

Comment:
can we rename to activeNodesToUnstake ?

Reply:
these are actualy the number of nodes which will be staked from the waiting. nodes were unstaked previously on line 325